libcnb-data/CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Changelog
 
 ## [Unreleased]
+
+## [0.6.0] 2022-04-08
```

Comment:
Whilst none of these changes are breaking (from my interpretation of the changes in #382 at least), it seemed preferable to not just do a "patch" release here, given it's a new feature addition. (Though technically anything goes pre v1 according to https://semver.org, so guess doesn't matter either way.)